flagging_site/blueprints/flagging.py

```diff
@@ -72,13 +72,57 @@ def index() -> str:
     returns: render model on index.html
     """
     df = get_data()
-    flags = {
-        2: reach_2_model(df, rows=1)['safe'].iloc[0],
-        3: reach_3_model(df, rows=1)['safe'].iloc[0],
-        4: reach_4_model(df, rows=1)['safe'].iloc[0],
-        5: reach_5_model(df, rows=1)['safe'].iloc[0]
+    
+    # Important: Restructured these dicts for easier use.
+    # Warning - The following aren't implemented yet due to inconsistencies with http://notification.crwa.org/wqmodel/charlesflags.php (old) vs https://www.crwa.org/flagging-program.html (new):
+        # not on old CRWA page, hence not implemented below:
+        # 'Northeastern's Henderson Boathouse'
+        # 'Paddle Boston at Herter Park' -- is this the same as Reach 3's 'CRCK at Herter Park?'
+
+        # only found on old CRWA page only, hence implemented according to old website's format:
+        # 'CRCK at Herter Park'
+        # 'CRCK at Kendall Square'
+
+    homepage = {
+        2: {
+            'flag': reach_2_model(df, rows=1)['safe'].iloc[0],
+            'time': reach_2_model(df, rows=25)['time'].iloc[0],
+            'names': {
+                0: 'Newton Yacht Club',
+                1: 'Watertown Yacht Club',
+                2: 'Community Rowing, Inc.'
+            }
+        },
+        3: {
+            'flag': reach_3_model(df, rows=1)['safe'].iloc[0],
+            'time': reach_3_model(df, rows=25)['time'].iloc[0],
+            'names': {
+                0: 'CRCK at Herter Park', # Found on old CRWA page only
+                1: 'Harvard\'s Weld Boathouse' 
+            }
+        },
+        4: {
+            'flag': reach_4_model(df, rows=1)['safe'].iloc[0],
+            'time': reach_4_model(df, rows=25)['time'].iloc[0],
+            'names': {
+                0: 'Riverside Boat Club'
+            }
+        },
+        5: {
+            'flag': reach_5_model(df, rows=1)['safe'].iloc[0],
+            'time': reach_5_model(df, rows=25)['time'].iloc[0],
+            'names': {
+                0: 'Charles River Yacht Club',
+                1: 'Union Boat Club',
+                2: 'Community Boating',
+                3: 'CRCK at Kendall Square', # Found on old CRWA page only
+                4: 'Paddle Boston at Kendall Square' # Assuming this is reach 5
+            }
```

Comment:
Make all of these lists instead of dicts.

Also, instead of `names`, call it `boathouses`. "Name" is vague (name of what, exactly?), but "boathouses" is not.

For example, rows 114-120 would be updated to:

```python
            'boathouses': [
                'Charles River Yacht Club',
                'Union Boat Club',
                'Community Boating',
                'CRCK at Kendall Square', # Found on old CRWA page only
                'Paddle Boston at Kendall Square' # Assuming this is reach 5
            ]
```

So, why lists and not dicts? The reason why is because a dict is a key-value store; it maps a key to a specific value. However, here the keys `0`, `1`, etc. have no special significance in this case. Charles River Yacht Club, for example, has no special significance in its relation to `0` other than the arbitrary choice to put it first and to give it a 0. You'd rather simply use a list in this case.

The list will preserve the order when being passed into Jinja, so don't worry about order. In fact, list order is safer to preserve than dict order (for example, in version of Python before 3.5, dicts were totally randomly ordered, whereas lists were not).

BTW, good work adding in those comment! I can definitely ask CRWA about those ones you're not sure of.